tools/documentation-generation/generate-documentation.js

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env node
+
+const fs = require("fs");
+const { Documentation, Language } = require("jsii-docgen");
+const path = require("path");
+
+const targetFolder = path.resolve(
+  __dirname,
+  "..",
+  "..",
+  "website",
+  "docs",
+  "cdktf",
+  "api-reference"
+);
+
+function replaceSquareBrackets(docs) {
```

Comment:
Could you add a single line comment with an example of what this does? I'm having a hard time understanding what it does by reading the code.